cfme/web_ui/paginator.py

```diff
@@ -78,28 +86,28 @@ def sort_by(sort):
     sel.select(Select(select), sel.ByText(sort))
 
 
+def _records():
+    matches = re.search(_records_re, _page_nums()).groupdict()
+    return {k: int(v) for k, v in matches.iteritems()}
```

Comment:
I'm explicitly turning the results into int here, where previously they were strings. I didn't see anything that would break as a result of this in the code, but wouldn't mind a double check.